Docker/Settings/DB-Admin/pgsql/Dockerfile

```diff
@@ -12,24 +12,21 @@ ENV POSTGRES_DB=postgres
 
 ARG VERSION=latest
 
-USER root
-RUN apk --no-cache add dos2unix=~7 unzip=~6
-
```

Comment:
The reason for putting the `apk` command above, in the base image, is for caching. It would be better to keep the command above, and just add `USER postgres` on line 17.